nestkernel/event.h

```diff
@@ -827,13 +843,20 @@ class GapJEvent : public SecondaryEvent
   static void
   set_syn_id( const synindex synid )
```

Comment:
Static methods can also be called on objects, and since both `set_syn_id()` and `add_syn_id()` only manipulate the static `supported_syn_ids_` member, we can remove `set_syn_id` and make `add_syn_id()` static.